Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "MIT"
 documentation = "https://docs.rs/scroll"
 description = "A suite of powerful, extensible, generic, endian-aware Read/Write traits for byte buffers"
 include = ["src/**/*", "Cargo.toml", "LICENSE", "README.md"]
+rust-version = "1.63"
```

Comment:
remove

Reply:
this is part of the #90 on which this PR was based.